processor/lib/azureBlob.ts

```diff
@@ -35,71 +35,71 @@ export class AzureBlob {
   }
 
   async ensureContainerExists(): Promise<CreateContainerResult> {
-    return await this._blobService.ensureContainerExists(this._containerName);
```

Comment:
If you're not gonna await, declaring these as non-async will save many bytes in the emitted ts code.

Reply:
Fixed. I remember I invoked a code fix in VS Code to fix all of these. I wonder why the code fix doesn't remove `async` as well.